apps/price_lists/src/pages/Home.tsx

```diff
@@ -57,10 +57,14 @@ export function Home(): JSX.Element {
         }
       />
     ) : (
-      <EmptyState
-        title='No price lists found!'
-        description='If you were looking for an SKU instead of a Price list, search again within "All Prices".'
-      />
+      <>
+        <Text weight='semibold'>
+          No results found. Try a new search.
+          <br />
+          If you were looking for an SKU instead of a Price list, search again
+          within "All Prices".
```

Comment:
is it possible to add a link to `All Prices`?

Reply:
Sure, done! Good idea!